packages/neuron-wallet/tests-e2e/env.ts

```diff
@@ -0,0 +1,23 @@
+export interface TransactionSendTo {
+  address: string
+  amount: number
+}
+
+export interface TransactionTestEnv {
+  // Used to test sending transactions, please make sure there is enough balance.
```

Comment:
Send txs requires connecting to a real CKB node. At the moment that might be too much work to setup. We can focus on other fundamental UI and features, and lower priority of on chain operations for now.

Reply:
It is designed to make it easier for testers to send multiple receive address transactions.